docs/projects/ad9265_fmc/index.rst

```diff
@@ -0,0 +1,230 @@
+     - FMC LPC
+
+Block design
+-------------------------------------------------------------------------------
+
+Block diagram
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. image:: ad9265_fmc_block_diagram.svg
+   :width: 800
+   :align: center
+   :alt: AD9265/ZedBoard block diagram
+
+Clock scheme
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+There are 3 ways to configure the clock source for :adi:`AD9265` (some 
+modification maybe necessary).
+
+- External passive clock (default)
+- Optional active clock path using the :adi:`AD9517`
+- Optional oscillator
+
+For more details, check :adi:`AD9265` schematic.
+
+
+CPU/Memory interconnects addresses
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The addresses are dependent on the architecture of the FPGA, having an offset
+added to the base address from HDL (see more at :ref:`architecture`).
+
+Depending on the values of parameters $INTF_CFG, $ADI_PHY_SEL and $TDD_SUPPORT,
+some IPs are instatiated and some are not.
+
+Check-out the table below to find out the conditions.
+
+==================== =============== 
+Instance             Zynq/Microblaze
+==================== =============== 
+axi_ad9265           0x44A00000         
+axi_ad9265_dma       0x44A30000     
+==================== =============== 
+
+SPI connections
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+THESE ARE JUST EXAMPLES!!!
+USE WHICHEVER FITS BEST YOUR CASE
+
+.. list-table::
+   :widths: 25 25 25 25
+   :header-rows: 1
+
+   * - SPI type
+     - SPI manager instance
+     - SPI subordinate
+     - CS
+   * - PS
+     - SPI 0
+     - AD9517
+     - 1
+   * - PS
+     - SPI 1
+     - AD0000
+     - 0
+
+Interrupts
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Below are the Programmable Logic interrupts used in this project.
+
+You have many ways of writing this table: as a list-table or really to draw
+it. Take a look in the .rst of this page to see how they're written and
+which suits best your case.
+
+.. list-table::
+   :widths: 30 10 15 15
+   :header-rows: 1
+
+   * - Instance name
+     - HDL
+     - Linux Zynq
+     - Actual Zynq
+   * - axi_ad9265_dma
+     - 13
+     - 57
+     - 89
+
+Building the HDL project
+-------------------------------------------------------------------------------
+
+The design is built upon ADI's generic HDL reference design framework.
+ADI does not distribute the bit/elf files of these projects so they
+must be built from the sources available :git-hdl:`here <>`. To get
+the source you must
+`clone <https://git-scm.com/book/en/v2/Git-Basics-Getting-a-Git-Repository>`__
+the HDL repository.
+
+Then go to the project location(**projects/ad9265_fmc/carrier**) and run the make command by
+typing in your command prompt(this example :adi:`ZC706``):
```

Comment:
Extra back tick, correct **format**:
```
:adi:`ZC706`
```
Note: `https://www.analog.com/ZC706.html` doesn't exist! Isn't a xilinx link that you want?

Reply:
Also add a space between "location" and the following parenthesis